src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1155,21 +1155,33 @@ export default class DefaultDeviceController
       return true;
     }
     const deviceIds = DefaultDeviceController.getIntrinsicDeviceId(device);
+    this.logger.debug(
+      `Checking deviceIds ${deviceIds} of type ${typeof deviceIds} with groupId ${groupId}`
```

Comment:
Could you ensure that you are not seeing the [Object object] for logging an array?

Reply:
Yes, i verified it. It printed out as devicesIds <list of deviceId delimited by comma>